src/coprocessor/endpoint.rs

```diff
@@ -296,7 +296,10 @@ impl Host {
                         future::ok::<_, futures_mpsc::SendError<_>>((resp, (ctx, finished)))
                     })
                 });
-                on_resp.respond_stream(box s, pool.clone());
+
+                let (tx, rx) = futures_mpsc::unbounded();
```

Comment:
Please use `bounded` queue. Otherwise if the client suspend, big requests may cause OOM.

Reply:
Shall we write a test? I'm not sure about what will happen when the client is suspended.